lib/Widgets/Settings.vala

```diff
@@ -14,6 +15,56 @@ namespace Granite {
      * Granite.Settings provides a way to share Pantheon desktop settings with applications.
      */
     public class Settings : Object {
+        /**
+         * Possible accent color preferences expressed by the user
+         */
+         public enum AccentColor {
```

Comment:
I'm fine with this names, they are easy to remember so, this is more of a question. Do we want to use `BLUE` instead of `BLUEBERRY`? The same applies for lime, banana, bubblegum, etc.

Let's see what the others think about it

Reply:
@cassidyjames has already thought about this 😀️

> @meisenzahl io.elementary.AccentColor seems fine to me. Could we ensure we're matching this mapping, though? I think we're also generally moving to only use the cutesy food names in user strings and just use more standard color names anywhere for developers.
> 
> [elementary/default-settings@17d737e](https://github.com/elementary/default-settings/commit/17d737ecc1fdd083127cbc290c49e3ad0563030b)

Reply:
Cool, makes sense :+1: